### PR TITLE
feat: upgrade module, add test instructions

### PR DESCRIPTION
**Pull request by laszlokr**

### CODE REVIEW

File: purchase_requisition_order_address/models/purchase_order.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+from odoo import api, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    partner_order_id = fields.Many2one(
```
Comment (janikvonrotz) [inline purchase_requisition_order_address/models/purchase_order.py]: This field and compute method might already be defined in `purchase_order_address`.

**Review by janikvonrotz — CHANGES_REQUESTED** [review]

File: purchase_requisition_order_address/tests/TEST_INSTRUCTIONS.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+- Open a contact for Deco Addict, set the Order Address
+- Open Purchase > Orders > Blanket Orders
+- Create a new Blanket Order for Deco Addict, the Order Address is copied from the contact
```
Comment (janikvonrotz) [inline purchase_requisition_order_address/tests/TEST_INSTRUCTIONS.rst:3]: Can you also test if the order address of the contract overules the order adress of the partner when creating a purchase order?

**Review by janikvonrotz — CHANGES_REQUESTED** [review]

File: purchase_requisition_order_address/tests/TEST_INSTRUCTIONS.rst (hunk 2 above)
Comment (laszlokr) [reply purchase_requisition_order_address/tests/TEST_INSTRUCTIONS.rst:3]: I updated the test instructions to reflect that it is carried from the Blanket order to the Purchase order, but maybe I don't fully understand your question. How does it overrule the order address of the partner? The field is readonly at each step, so only the order address set on the contract can be used, and can only be changed in the contact itself

Comment (janikvonrotz) [reply purchase_requisition_order_address/tests/TEST_INSTRUCTIONS.rst:3]: The field should be writable on every step.

If somebody changes the order partner on the requisition and creates an order, the order should copy the order partner from the requisition.

**Review by janikvonrotz — CHANGES_REQUESTED** [review]
Make the field writable.

File: purchase_requisition_order_address/tests/TEST_INSTRUCTIONS.rst (hunk 2 above)
Comment (laszlokr) [reply purchase_requisition_order_address/tests/TEST_INSTRUCTIONS.rst:3]: @janikvonrotz currently, the field can be changed on the blanket order and RFQ, but not on the confirmed purchase order which has a delivery. This is the correct behavior, right?

Comment (laszlokr) [reply purchase_requisition_order_address/tests/TEST_INSTRUCTIONS.rst:3]: @janikvonrotz I also made it writable here: https://github.com/Mint-System/Odoo-Apps-Purchase-Workflow/pull/2/commits/69a6bc12bd5268772b098352f5b743a61f16e861

Comment (janikvonrotz) [reply purchase_requisition_order_address/tests/TEST_INSTRUCTIONS.rst:3]: @laszlokr Yes, once its confirmed it is not writable.

You can check it on the production system: https://erp.trimada.ch/

**Comment by laszlokr** [comment]
@janikvonrotz can you approve the PR or is there something else I need to resolve?

**Review by janikvonrotz — APPROVED** [review]

**Pull request merged**